po/ru.po

```diff
@@ -855,12 +856,12 @@ msgstr "Переинициализировать sdk/окружение"
 #: app/flatpak-builtins-build-init.c:118
 #, fuzzy, c-format
```

Comment:
This string was updated but still marked as fuzzy.

Reply:
Well, let it remain as fuzzy, it doesn’t affect the translation itself in any way

Reply:
It does. Fuzzy translations are omitted from compiled translation files.

Reply:
> It does. Fuzzy translations are omitted from compiled translation files.

I agree, but it does not affect the translation itself. Can be translated or NOT translated

Reply:
Well, we translate programs because we want the translated text in the program, right? Otherwise, what's the point of translation if it's not used.

Reply:
Hmm... or maybe it’s like this: today it’s fuzzy, and tomorrow it’s taken and ceased to be fuzzy

Reply:
This wasn't addressed. Please mark ready translations as ready.

Reply:
All is in readiness